CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+- Added App Hosting as an option for firebase init. (#7803)
```

Comment:
this is part of the previous release https://github.com/firebase/firebase-tools/releases/tag/v13.22.0
```suggestion
```